grakn-dist/src/bin/grakn-engine.sh

```diff
@@ -48,9 +48,9 @@ start)
         # engine has not already started
         echo -n "Starting engine"
         if [[ "$FOREGROUND" = true ]]; then
-            java -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}/bin" -Dgrakn.conf="${GRAKN_ENGINE_CONFIG}" ai.grakn.engine.GraknEngineServer
+            java -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}/bin" -Dgrakn.conf="${GRAKN_CONFIG}" ai.grakn.engine.GraknEngineServer
```

Comment:
We've got a separate pull request in for these startup script changes - could we remove them from this one? Just to try and keep things delineated.